### PR TITLE
add radial motion

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: src/dot/dotMotionSimulation.m [hunk 1]
```diff
@@ -0,0 +1,105 @@
+function relativeDensityContrast = dotMotionSimulation(cfg, thisEvent, nbEvents, doPlot)
```
Comment (marcobarilari) [inline src/dot/dotMotionSimulation.m:1]: mind blowing, awsome!!!

File: src/dot/dotMotionSimulation.m [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    relativeDensityContrast = (max(dotDensity(:)) - min(dotDensity(:))) / max(dotDensity(:));
+
+    if doPlot
+        imagesc(dotDensity);
+        axis square;
+        title('dot density');
+    end
+
+    fprintf(1, '\n');
```
Comment (marcobarilari) [inline src/dot/dotMotionSimulation.m:84]: what's that?

**Comment by marcobarilari** [comment]
very nice, LGTM and ready to be merged. I don't suggest to add stuff in the docs cause some automation will come (right?)

**Review by marcobarilari — APPROVED** [review]

**Comment by marcobarilari** [comment]
btw, is cpp-lln-lab/localizer_visual_motion#4 fixed this pr?

**Comment by Remi-Gau** [comment]
> btw, is [cpp-lln-lab/localizer_visual_motion#4](https://github.com/cpp-lln-lab/localizer_visual_motion/issues/4) fixed by this pr?

yes this should be a bit of a hacky fix to this issue.

not super clean but at least we have now a way to vizualize the result

**Comment by Remi-Gau** [comment]
> very nice, LGTM and ready to be merged. I don't suggest to add stuff in the docs cause some automation will come (right?)

hopefully this should come soon !

**Pull request merged**